internal/blocktx/store/postgresql/get_stale_chain.go

```diff
@@ -47,7 +45,6 @@ func (p *PostgreSQL) GetStaleChainBackFromHash(ctx context.Context, hash []byte)
 				,b.merkleroot
 				,b.height
 				,b.processed_at
-				,b.orphanedyn
 				,b.status
 				,b.chainwork
 			FROM blocktx.blocks b JOIN prevBlocks p ON b.hash = p.prevhash AND b.status = $2
```

Comment:
You could stop execution on first non-processed ancestor

Reply:
Done, in most of other queries as well